shuup/core/signal_handlers.py

```diff
@@ -65,6 +68,19 @@ def handle_contact_post_save(sender, instance, **kwargs):
     bump_internal_cache()
 
 
+@receiver(order_creator_finished)
+def on_order_creator_finished(sender, order, source, **kwargs):
+    # reset product prices
+    for product_id, shop_id in order.lines.exclude(product__isnull=False).values_list("product_id", "order__shop_id"):
+        context_cache.bump_cache_for_product(product_id, shop_id)
+
+
+@receiver(order_changed)
+def on_order_changed(sender, order, **kwargs):
```

Comment:
Should these be connected same way as the other ones so we have option to disconnect and perhaps handle the bumping in some other way?

Reply:
Ok dispatch uid is not needed for disconnecting the signal.